doc/rst/developer/bestPractices/CHPL_DEVELOPER.rst

```diff
@@ -8,7 +8,7 @@ e.g. ``true``).  In particular, all code that is committed should be
 able to be built with ``CHPL_DEVELOPER`` is set; otherwise, that code is
 considered to break the build.
 
-Setting ``CHPL_DEVELOPER`` has a number of impacts on how things behave,
+Setting ``CHPL_DEVELOPER`` has several impacts on how things behave,
```

Comment:
I'd rather not make the changes in this file, but I'm open to hearing your thoughts behind doing them.

Reply:
In my view this change makes the sentence precise and short, the 3 words can be replaced by a single word. No other reason though :sweat_smile:. I will do as you suggest.

Reply:
I think the uncertainty is more accurate - while CHPL_DEVELOPER shouldn't impact the behavior of a program, it does turn on a number of things that are only useful to a developer (log ids, better error messages for things that a user shouldn't trigger, compiler debugging settings, to name just a few).  I don't think distinguishing between whether the changes are "several" or "many" is particularly useful (but wouldn't be surprised if they could be qualified as "many", which is why I'm hesitant to say "several").

Reply:
Yes, agreed. Reverting the changes,